sway/desktop/xwayland.c

```diff
@@ -388,12 +388,21 @@ static void get_geometry(struct sway_view *view, struct wlr_box *box) {
 	}
 }
 
+static void handle_cache(struct wl_listener *listener, void *data) {
+	struct sway_xwayland_view *xwayland_view =
+		wl_container_of(listener, xwayland_view, cache);
+	struct sway_view *view = &xwayland_view->view;
+	struct wlr_xwayland_surface *xsurface = view->wlr_xwayland_surface;
+	struct wlr_surface_state *cached = data;
+	transaction_notify_view_acked_by_geometry(view,
```

Comment:
Xwayland surfaces don't have `ack_configure` event, so we notify the transaction system here. Currently this is done incorrectly, as `xsurface->{x,y}` isn't cached. In the future, this should be changed to iterating over `cached->addons`, looking for corresponding `wlr_xwayland_surface` state, and taking the positing from there.